recipes/gklib/build.sh

```diff
@@ -0,0 +1,13 @@
+mkdir -p build
+
+pushd build
+
+cmake -DCMAKE_INSTALL_PREFIX=$PREFIX \
+      -DCMAKE_PREFIX_PATH=$PREFIX \
+      -DDEBUG=OFF \
+      -DOPENMP=set \
+      -DBUILD_SHARED_LIBS=OFF \
```

Comment:
Conda-forge has a strong preference for shared libraries to make it easier to manage updates and migrations for new ABIs (it also makes license compliance simplier).

Is it practical here? Or is there a reason to why static linking is really needed?

Reply:
I think it would be practical on linux + OSX, but currently not on WIN as the `__dllexport/__dllimport` statements have not been properly set-up.

Should we package shared libraries for linux + OSX and static for WIN, or is it preferable to be consistent and keep everything static until everything can be built as shared?

Reply:
Keep as much as shared as possible. You should get a working shared library on Windows by setting `cmake -D CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS=ON`.

Reply:
@xhochy unfortunately the built binaries rely on a couple of global constants so `CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ` won't directly work here without patching.

Reply:
Oh, amazing! `CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS` hasn't failed for me before.

Reply:
Shared Unix builds are enabled in 1342d88.